llvm_jni/build.gradle

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Can we also get gradle to run the jar it creates and compile all the PVO*.bal files in the testSuite to .ll?

Reply:
Currently if you run `./gradlew testBuild` it will create a new create a new directory in the root called `testbuild` which
1. Has every thing in compiler except `nback.llvm`
2. Replace all the imports to `nback.llvm` with `jni.llvm` and add the `jni.llvm` modules code to that directory
3. Setup dependencies and run `bal build` in that directory.
I think step 3 covers this or should it do something in addition to that to run those tests

Reply:
`bal build` just checks that the compilation of the files in testSuite does not return an error.

We also need to check that the generated .ll files are correct. One way is by comparing to the .ll files we generated through nback.llvm (perhaps using `llvm-diff`). Another is by compiling and executing.